src/software/ai/hl/stp/play/test_plays/robot_substitution_test.cpp

```diff
@@ -0,0 +1,71 @@
+
+TEST_F(TacticAssignmentTest, test_assigning_stop_tactics_to_unassigned_non_goalie_robots)
```

Comment:
why did you make a new test file and not just put it here: https://github.com/UBC-Thunderbots/Software/blob/master/src/software/ai/hl/stp/stp_tactic_assignment_test.cpp